app/web/src/components/HtmlMeta.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  HTML_META_DESCRIPTION,
+  HTML_META_TITLE,
+  HTML_META_TITLE_SUFFIX,
+} from "features/constants";
+import { Helmet } from "react-helmet";
+
+interface HtmlMetaProps {
+  title?: string;
+  sharingTitle?: string;
+  description?: string;
+  noSuffix?: boolean;
+}
+
+export default function HtmlMeta(props: HtmlMetaProps) {
+  const title = props.title || HTML_META_TITLE;
+  const sharingTitle = props.sharingTitle || title;
+  const description = props.description || HTML_META_DESCRIPTION;
```

Comment:
Just stylistic thing, but in the rest of the repo we destructure the props in the function definition and use defaults, like this:
`export default function HtmlMeta({firstProp = DEFAULT_VALUE, secondProp = X}: HtmlMetaProps) {`